osu.Game/Overlays/Comments/CommentsContainer.cs

```diff
@@ -117,41 +138,93 @@ public CommentsContainer(CommentableType type, long id)
         private void load()
         {
             background.Colour = colours.Gray2;
+            placeholderBackground.Colour = colours.Gray3;
         }
 
         protected override void LoadComplete()
         {
-            Sort.BindValueChanged(onSortChanged, true);
+            Sort.BindValueChanged(_ => updateComments());
+            api.LocalUser.BindValueChanged(_ => updateComments());
```

Comment:
Just noticed - isn't this a memory leak? You should probably make a local bindable and bind to `api.LocalUser`.